lua/treesitter-context.lua

```diff
@@ -125,7 +111,15 @@ function M.enable()
 
   attached[cbuf] = true
 
-  autocmd({ 'WinScrolled', 'BufEnter', 'WinEnter', 'VimResized' }, update)
+  local update_events = {
+    'WinScrolled',
+    'BufEnter',
+    'WinEnter',
+    'VimResized',
+    'CursorMoved',
+  }
+
+  autocmd(update_events, update)
```

Comment:
Again, another unnecessary change that distracts from reviewing the rest of the code.